packages/remix-node/upload/fileUploadHandler.ts

```diff
@@ -59,8 +59,8 @@ export type FileUploadHandlerOptions = {
   /**
    *
    * @param filename
-   * @param mimetype
-   * @param encoding
+   * @param contentType
+   * @param name
```

Comment:
It would be better to fix the JSDOC 

```
   * @param FileUploadHandlerFilterArgs <desc>
   * @param FileUploadHandlerFilterArgs.filename <desc>
   * @param FileUploadHandlerFilterArgs.mimetype <desc>
   * @param FileUploadHandlerFilterArgs.encoding <desc>
```

Reply:
Not sure what you mean @3imed-jaberi, the accurate properties are contentType and name.

Reply:
@machour please check [this JSDOC ref #Documenting a parameter's properties](https://jsdoc.app/tags-param.html)

Reply:
@3imed-jaberi oh yes I see. I'm sticking to the team's choice here. 
Feel free to open a new discussion to propose a change to jsdoc all over the repo.